src/python/pants/option/options_bootstrapper.py

```diff
@@ -192,7 +193,7 @@ def is_bootstrap_option(arg: str) -> bool:
         # Take just the bootstrap args, so we don't choke on other global-scope args on the cmd line.
         # Stop before '--' since args after that are pass-through and may have duplicate names to our
         # bootstrap options.
-        bargs = ("./pants",) + tuple(
+        bargs = (bin_name(),) + tuple(
```

Comment:
This one is weird, but works because of the fallback.

Reply:
If you take my recommendation to raise KeyError, then we should probably go back to using `./pants` with a `# noqa`. I don't think it matters what this first value is iirc

Reply:
If it doesn't matter, I'd suggest putting in `"<ignored>"` or some such.. ;)